pandas/tests/reshape/merge/test_merge_asof.py

```diff
@@ -588,14 +590,23 @@ def test_non_sorted(self):
         # ok, though has dupes
         merge_asof(trades, self.quotes, on="time", by="ticker")
 
-    def test_tolerance(self):
+    @pytest.mark.parametrize(
+        "tolerance",
+        [
+            Timedelta("1day"),
+            pytest.param(
+                datetime.timedelta(days=1),
+                marks=pytest.mark.xfail(reason="not implemented", strict=True),
```

Comment:
Do we have an open issue to implement this?

Reply:
No, not that I know of.